src/components/Button/Button.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
```

Comment:
We somehow ended up with both jsx and tsx versions of this Component. Any improvements to Button should be salvaged from this file, and it should be trashed.

Reply:
agree, remove